src/configure/clients/github/TemplateServiceClient.ts

```diff
@@ -28,4 +28,23 @@ export class TemplateServiceClient {
             '6.0-preview.1'
         );
     }
+
+    public async getTemplateConfiguration(templateId: string, inputs: StringMap<string>): Promise<ExtendedPipelineTemplate> {
+        let requestUri = this.templateServiceUri + "/" + templateId + "/configuration";
+        return this.restClient.sendRequest2(
+            requestUri,
+            'POST',
+            '6.0-preview.1',
+            inputs
+        );
+    }
+
+    public async getTemplateFile(templateId: string, fileId: string): Promise<string> {
+        let requestUri = this.templateServiceUri + "/" + templateId + "/files" + fileId;
```

Comment:
Does fileId contain a '/' at start? Should we put one here explicitly as well to ensure that url does not break

Reply:
done